WNPRC_Purchasing/src/client/components/PurchasingFormInput.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import {Col, Row} from 'react-bootstrap';
+
+interface PurchasingFormInputProps
+{
+    label: string;
+    required?: boolean;
+    colSize?: number;
+}
+
+export class PurchasingFormInput extends React.PureComponent<PurchasingFormInputProps, any>
```

Comment:
Might be good to be consistent with using FC with hooks or component classes.  At least within the same project

Reply:
good point, now done